### PR TITLE
Add server roles info embed

### PR DESCRIPTION
**Pull request by Soheab**
- Added suggested server roles info with descriptions.
- Added a hyperlink to queue embed like we discussed.

Please review all the descriptions and leave a comment on the ones that need to be improved. Also see the comments in code.

### CODE REVIEW

File: cogs/admin.py [hunk 1]
```diff
@@ -317,11 +350,14 @@ async def rulesninfo(self, ctx):
         channel = ctx.guild.get_channel(716717317320605736)
         server_rules = await channel.fetch_message(723643619315023873)
         bot_rules = await channel.fetch_message(723643619700899983)
+        #server_roles = await channel.fetch_message(ID) <- uncomment with message id
```
Comment (GavynStanley) [inline cogs/admin.py]: Do it yourself

File: cogs/admin.py [hunk 2]
```diff
@@ -317,11 +350,14 @@ async def rulesninfo(self, ctx):
...
         links = await channel.fetch_message(723643620313268291)
         faqs = await channel.fetch_message(723643620946870272)
 
         await server_rules.edit(embed = server_rules_embed)
         await bot_rules.edit(embed = bot_rules_embed)
+        await channel.send(embed = server_roles_embed)  # delete this
+        #await server_roles.edit(embed = server_roles_embed) <- uncomment
```
Comment (GavynStanley) [inline cogs/admin.py]: Do it yourself

**Review by GavynStanley — CHANGES_REQUESTED** [review]

File: cogs/admin.py (hunk 1 above)
Comment (Soheab) [reply cogs/admin.py]: i can't run the command for the first time and then edit code without another pr

File: cogs/admin.py (hunk 2 above)
Comment (Soheab) [reply cogs/admin.py]: i can't run the command for the first time and then edit code without another pr

**Comment by GavynStanley** [comment]
I'm dumb, please ignore me

**Review by GavynStanley — APPROVED** [review]

**Review by GavynStanley — APPROVED** [review]

**Pull request merged**